python/ray/serve/tests/test_telemetry.py

```diff
@@ -484,5 +484,104 @@ def test_lightweight_config_options(manage_ray, lightweight_option, value):
             assert tagkey not in report["extra_usage_tags"]
 
 
+@pytest.mark.parametrize("location", ["driver", "deployment"])
+def test_status_api_detected(manage_ray, location):
+    """Check that serve.status is detected correctly by telemetry."""
+
+    subprocess.check_output(["ray", "start", "--head"])
+    wait_for_condition(check_ray_started, timeout=5)
+
+    storage_handle = start_telemetry_app()
+    wait_for_condition(
+        lambda: ray.get(storage_handle.get_reports_received.remote()) > 0, timeout=5
+    )
+
+    if location == "deployment":
+
+        @serve.deployment
+        class Model:
+            async def __call__(self):
+                return serve.status()
+
+        handle = serve.run(Model.bind(), route_prefix="/model")
+        handle.remote()
+    elif location == "driver":
+        serve.status()
+
+    def check_telemetry():
```

Comment:
Non-blocker: I think it would be nice to check those telemetries doesn't exist before the api calls just to double check (pretty sure it will pass as well). Same goes to other ones.

Reply:
Makes sense, I've added it as a test case to all 3 tests!